align/pnr/main.py

```diff
@@ -25,13 +26,18 @@ def _generate_json( *, dbfile, variant, primitive_dir, pdk_dir, output_dir, chec
         ncpy = int(m.groups()[1])
         assert ncpy == hN.n_copy, f"n_copy {hN.n_copy} should be same as in the variant name {variant} {ncpy}"
 
-    res = gen_viewer_json( hN, pdkdir=pdk_dir, draw_grid=True, json_dir=str(primitive_dir), checkOnly=(check or extract), extract=extract, input_dir=input_dir, toplevel=toplevel)
+    res = gen_viewer_json( hN, pdkdir=pdk_dir, draw_grid=True, json_dir=str(primitive_dir), checkOnly=(check or extract or gds_json), extract=extract, input_dir=input_dir, toplevel=toplevel)
 
-    if check or extract:
+    if check or extract or gds_json:
         cnv, d = res
     else:
         d = res
 
+    if gds_json and toplevel:
+        # Hack in Outline layer
+        # Should be part of post processor
+        d['terminals'].append( { "layer": "Outline", "netName": None, "rect": d['bbox']})
```

Comment:
@parijatm We should move this somewhere else. I guess we have a global postprocessor hook.